process/templates/SIG_NAME_graduation_stage.md

```diff
@@ -1,5 +1,32 @@
 ## Special Interest Group (SIG) graduation
 
 The governing body must agree that the SIG has completed a major deliverable
-
   * Link to relevant documentation.
+
+## SIG has met all Incubating requirements
+  * "link to Incubating PR"
+
+### Governance
+SIG must have documented project governance and be able to demonstrate that governance in action.
+  * "link to governance documents"
+SIG has met at least 4 times over a period of at least 2 months since becoming incubating
+  * "link to meeting agenda"
+Have a define roadmap and annual goals for the project
+  * "link to roadmap and goals"
+SIG has completed a major deliverable
+	  * "link to completed deliverable"
+
+### To retain Graduated status
+A TI is expected to continue operating per the above guidelines, and to provide the TAC with quarterly status updates, and to approach the TAC when seeking approval of substantial changes (such as when accepting or promoting new Projects).
```

Comment:
These requirements are appropriate for projects/WGs, but I wonder if this makes sense for SIGs. I suggest, we adjust this to more closely match the language from the [SIG lifecycle doc](https://github.com/ossf/tac/blob/main/process/sig-lifecycle.md#once-graduated).
```suggestion
### Once in Graduated Status

The SIG should continue to update the state of the SIG on its README as further work is performed, or make the decision to transition to [Archived](https://github.com/ossf/tac/blob/main/process/sig-lifecycle.md#to-become-archived).
```